borg/archive.py

```diff
@@ -589,7 +572,10 @@ def process_file(self, path, st, cache, ignore_inode=False):
                 status = 'U'  # regular file, unchanged
         else:
             status = 'A'  # regular file, added
-        item = {b'path': safe_path}
+        item = {
+            b'path': safe_path,
+            b'link': st.st_nlink > 1,
```

Comment:
so the meaning of "link" is that this item is member of a set of hardlinks (the first one will have data, the other members will refer to the first one). maybe this should be in a comment in the src.